Gemfile.lock

```diff
@@ -271,6 +271,7 @@ GEM
 PLATFORMS
   arm64-darwin-20
   arm64-darwin-21
+  ruby
```

Comment:
Wait, wut?

Reply:
yeah, I was surprised by this too, but it was done automatically so I figured it was probably fine

Reply:
apparently bundler 2.1 does this https://www.moncefbelyamani.com/understanding-the-gemfile-lock-file/

Reply:
just tried upgrading to the latest bundler and doing `bundle install` again and it made no changes, so I'm guessing this is fine. 

I'm going to merge and release for now, since this is also just for us (and our own usage of fastlane), but I'm happy to open up a new PR to remove it if you feel it's needed.